[2025-03-16 06:49:55,638] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_two 2024-11-27T00:00:00+00:00 [queued]>
[2025-03-16 06:49:55,656] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_two 2024-11-27T00:00:00+00:00 [queued]>
[2025-03-16 06:49:55,656] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2025-03-16 06:49:55,656] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2025-03-16 06:49:55,657] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2025-03-16 06:49:55,666] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): task_two> on 2024-11-27T00:00:00+00:00
[2025-03-16 06:49:55,670] {standard_task_runner.py:52} INFO - Started process 182 to run task
[2025-03-16 06:49:55,673] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'task_two', '2024-11-27T00:00:00+00:00', '--job-id', '43', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmpqsc_f8he', '--error-file', '/tmp/tmp_wpqhdmh']
[2025-03-16 06:49:55,674] {standard_task_runner.py:77} INFO - Job 43: Subtask task_two
[2025-03-16 06:49:55,713] {logging_mixin.py:104} INFO - Running <TaskInstance: assignment.task_two 2024-11-27T00:00:00+00:00 [running]> on host 4ecec8cb0ca2
[2025-03-16 06:49:55,762] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=assignment
AIRFLOW_CTX_TASK_ID=task_two
AIRFLOW_CTX_EXECUTION_DATE=2024-11-27T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-27T00:00:00+00:00
[2025-03-16 06:49:55,764] {python.py:151} INFO - Done. Returned value was: None
[2025-03-16 06:49:55,776] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=assignment, task_id=task_two, execution_date=20241127T000000, start_date=20250316T064955, end_date=20250316T064955
[2025-03-16 06:49:55,798] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16 06:49:55,807] {local_task_job.py:151} INFO - Task exited with return code 0
